airbyte-cdk/python/airbyte_cdk/sources/declarative/manifest_declarative_source.py

```diff
@@ -66,7 +73,9 @@ def connection_checker(self) -> ConnectionChecker:
         check = self._source_config["check"]
         if "type" not in check:
             check["type"] = "CheckStream"
-        check_stream = self._constructor.create_component(CheckStreamModel, check, dict())
+        check_stream = self._constructor.create_component(
+            CheckStreamModel, check, dict(), emit_connector_builder_messages=self._emit_connector_builder_messages
```

Comment:
pass the field as argument since the `ModelToComponentFactory` doesn't know about it